data/pizza.json

```diff
@@ -1069,7 +1069,7 @@
 			"hidden_ingredients": [
 				"bröd"
 			],
-			"price": 90
+			"price": 95
 		}
 	},
 	"hamburgers": {
```

Comment:
[No ost on the Amerikansk salladsburgare anymore](https://sannegarden.com/meny/johanneberg/#:~:text=85Kr-,Skinka%2C%20Ananas%2C%20Gurka%2C%20Isbergssallad%2C%20Rhode%20Islands%C3%A5s%2C%20Majs%2C%20Tomater,-Kyckling%2Dsalladsburgare)  
[And Rhode Islandsås on the Kyckling-salladsburgare](https://sannegarden.com/meny/johanneberg/#:~:text=Champinjoner%2C%20L%C3%B6k%2C%20Kyckling%2C%20Gurka%2C%20Isbergssallad%2C%20Rhode%20Islands%C3%A5s%2C%20Majs%2C%20Tomater)